### ISSUE TITLE
Create automated axe accessibility tests

### ISSUE THREAD

**Issue by edhamiltonHO** [HO-CTO/engineering-guidance-and-standards#132]
as part of #60 

- [x] Add and define tests
- [ ] Set tests to run as part of automated checks on a PR
- [ ] Run tests and raise issues for fixes where required

**Comment by edhamiltonHO** [comment]
Fixes #131  #130  #133  #126   to resolve issues raised by initial aXe test run

**Comment by edhamiltonHO** [comment]
To include a decision on the log to reflect that we have agreed:

- we will not fail the build based on negative aXe result relating to breadcrumbs or phase banner being outside landmark region
- issue with product page main banner being outside region is being resolved upstream
- we will review the aXe landmark failures when we update page layouts more comprehensively

as such, closing these issues with no changes
#133 
#130
